calico-cloud/visibility/elastic/flow/aggregation.mdx

```diff
@@ -10,96 +10,63 @@ Configure flow log aggregation level to reduce log volume and costs.
 
 ### Aggregation types and levels
 
-The following table summarizes the aggregation levels by flow log traffic.
-
-| Flow log aggregation by... | Available for...                                | Aggregates all flows that share...                                                                                                                                                                                                                                                                     |
-| -------------------------- | ----------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| Pod prefix                 | Allowed and denied traffic (default is allowed) | `FlowLogsFileAggregationKindForAllowed`<br />**0**, No aggregation <br />**1**, A source port on each node.<br />**2, (default)** Source ports, or are from the same ReplicaSet. <br />**3**, Destination and source ports, and are from the same ReplicateSet.                                        |
-| Source port                | Allowed and denied traffic                      | `FlowLogsFileAggregationKindForAllowed`<br/>`FlowLogsFileAggregationKindForDenied`<br />**0**, No aggregation<br />**1, (default)** Source port on each node.<br />**2**, Source ports, or are from the same ReplicaSet.<br />**3**, Destination and source ports, and are from the same ReplicateSet. |
+For allowed flows, the default aggregation level is 2, `AnyProcessInSamePodPrefix` and for denied flows the default aggregation level is 1,
+`AnyProcessInSameSourcePod`.
+
+The following table summarizes the aggregation levels by flow log traffic:
+
+| **Level** | **Name**                            | **Description**                                                   |
+|-----------|-------------------------------------|-------------------------------------------------------------------|
+| 0         |                                     | No aggregation                                                    |
+| 1         | AnyProcessInSameSourcePod           | Identity fields below source pod level are masked out. It means that if multiple processes or containers, within the same source pod, perform the same operation, the events are aggregated.       |
+| 2         | AnyProcessInSameSourcePodPrefix     | In addition to the above, source pod names are aggregated based on their shared prefixes. This means that flows, to the same destination, from pods within the same Deployment/ReplicaSet are aggregated together. |
+| 3         | AnyProcessInSamePodPrefix           | This level of aggregation builds on the previous two levels and also groups destination pod names based on their shared prefixes. |
```

Comment:
This feels like maybe going too far in copying the RS doc.  But I see that it does broadly work, and in particular that `AnyProcessInSameSourcePod` does correspond to source port masking.  That said:

- Would something like `AnyConnectionFromSameSourcePod` be better in this context than `AnyProcessInSameSourcePod`, to put the focus more on network connections than on processes?  (Although it is correct that connections have to made by processes...)  Then Level 3 could be `AnyConnectionBetweenSamePodPrefixes`.  (Although also need to incorporate Shaun's point here about not saying "pod prefix" if possible.)

- Similarly, perhaps "make the same connection to a destination" instead of "perform the same operation"?

Reply:
(I see you've addressed the point about "pod prefix" below.)

Reply:
I'm finding it hard to replacing `pod prefix` without creating a very long name. I feel the items have to be read along with their description.

I could replace:
`AnyConnectionFromSamePodPrefix` with `AnyConnectionFromSamePodController`
`AnyConnectionBetweenSamePodPrefixes` with `AnyConnectionBetweenSamePodControllers`

but I feel we're just replacing the word, without applying the intended change. WDYT?

Reply:
IMO, the words you already added to explain pod prefix suffice.  (I.e. the "In Kubernetes ..." paragraph.)  Given that you've done that, I would say it's OK to keep using `PodPrefix` here.